<launch>

   <rosparam file="$(find drone_contest_2022)/config/planner_setpoints.yaml"  />
   <node pkg="drone_contest_2022" name="interactive_planner" type="interactive_planner" output="screen" />

</launch>